build.sh

```diff
@@ -4,7 +4,7 @@ set -x
 
 ECR_PATH="public.ecr.aws/uktrade"
 
-BUILDER_VERSION="0.2.326-full"
+BUILDER_VERSION="${BUILDER_VERSION:-0.2.326-full}"
```

Comment:
Would this not be like this?

```suggestion
BUILDER_VERSION="${BUILDER_VERSION:=0.2.326-full}"
```

Reply:
@gabelton is correct. its :-  not := in bash

Reply:
If the variable is not set or null, then `:=` is better to actually assign the default value to the variable rather than substitution, no? Or I might be misunderstanding something here.

Reference: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
acording to the docs you are right.  I didnt think it mattered if the VAR was set, but doc says it does.  So lets go with :=